addons/collectd/0.0.1/Manifest

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
On CentOS this will install just the rpms for collectd and yajl - less than 1 MB altogether.

On Ubuntu it's installing 88MB of packages with the apt command we currently use to build and includes packages like mysql-common and perl-base. If we update the command to add `--no-install-recommends`, that gets the total down to 38 packages and 15MB and we're less likely to break the host.
```
-rw-r--r-- 1 root root  891K Mar  9  2020 collectd-core_5.7.2-2ubuntu1.2_amd64.deb
-rw-r--r-- 1 root root   20K Mar  9  2020 collectd_5.7.2-2ubuntu1.2_amd64.deb
-rw-r--r-- 1 root root   55K Apr  6  2018 fontconfig-config_2.12.6-0ubuntu2_all.deb
-rw-r--r-- 1 root root  165K Apr  6  2018 fontconfig_2.12.6-0ubuntu2_amd64.deb
-rw-r--r-- 1 root root 1017K Aug 17  2016 fonts-dejavu-core_2.37-1_all.deb
-rw-r--r-- 1 root root   41K Jan 20  2020 libbsd0_0.8.7-1ubuntu0.1_amd64.deb
-rw-r--r-- 1 root root  567K Jan 22  2019 libcairo2_1.15.10-2ubuntu0.1_amd64.deb
-rw-r--r-- 1 root root   13K Nov 27  2017 libcap2_1%3a2.25-1.2_amd64.deb
-rw-r--r-- 1 root root   18K Feb 14  2018 libdatrie1_0.2.10-7_amd64.deb
-rw-r--r-- 1 root root   27K Nov  1  2017 libdbi1_0.9.0-5_amd64.deb
-rw-r--r-- 1 root root   79K Sep 12  2019 libexpat1_2.2.5-3ubuntu0.2_amd64.deb
-rw-r--r-- 1 root root  134K Apr  6  2018 libfontconfig1_2.12.6-0ubuntu2_amd64.deb
-rw-r--r-- 1 root root  328K Oct 20 10:13 libfreetype6_2.8.1-2ubuntu2.1_amd64.deb
-rw-r--r-- 1 root root  1.2M Mar 24  2020 libglib2.0-0_2.56.4-0ubuntu0.18.04.6_amd64.deb
-rw-r--r-- 1 root root   77K Mar 11  2018 libgraphite2-3_1.3.11-2_amd64.deb
-rw-r--r-- 1 root root  227K Apr 13  2018 libharfbuzz0b_1.7.2-1ubuntu1_amd64.deb
-rw-r--r-- 1 root root  7.7M Mar 17  2020 libicu60_60.2-3ubuntu3.1_amd64.deb
-rw-r--r-- 1 root root   38K Nov  2  2016 libltdl7_2.4.6-2_amd64.deb
-rw-r--r-- 1 root root  149K Aug 22  2018 libpango-1.0-0_1.40.14-1ubuntu0.1_amd64.deb
-rw-r--r-- 1 root root   21K Aug 22  2018 libpangocairo-1.0-0_1.40.14-1ubuntu0.1_amd64.deb
-rw-r--r-- 1 root root   33K Aug 22  2018 libpangoft2-1.0-0_1.40.14-1ubuntu0.1_amd64.deb
-rw-r--r-- 1 root root  224K Dec 17  2017 libpixman-1-0_0.34.0-2_amd64.deb
-rw-r--r-- 1 root root  172K Apr 30  2019 libpng16-16_1.6.34-1ubuntu0.18.04.2_amd64.deb
-rw-r--r-- 1 root root  155K Mar  1  2018 librrd8_1.7.0-1build1_amd64.deb
-rw-r--r-- 1 root root  130K Feb 16  2018 libthai-data_0.1.27-2_all.deb
-rw-r--r-- 1 root root   18K Feb 16  2018 libthai0_0.1.27-2_amd64.deb
-rw-r--r-- 1 root root  558K Sep  2 13:03 libx11-6_2%3a1.6.4-3ubuntu0.3_amd64.deb
-rw-r--r-- 1 root root  111K Sep  2 13:03 libx11-data_2%3a1.6.4-3ubuntu0.3_all.deb
-rw-r--r-- 1 root root  7.4K Jun 19 06:08 libxau6_1%3a1.0.8-1ubuntu1_amd64.deb
-rw-r--r-- 1 root root   15K Mar  1  2019 libxcb-render0_1.13-2~ubuntu18.04_amd64.deb
-rw-r--r-- 1 root root  5.5K Mar  1  2019 libxcb-shm0_1.13-2~ubuntu18.04_amd64.deb
-rw-r--r-- 1 root root   45K Mar  1  2019 libxcb1_1.13-2~ubuntu18.04_amd64.deb
-rw-r--r-- 1 root root   11K Apr 30  2017 libxdmcp6_1%3a1.1.2-3_amd64.deb
-rw-r--r-- 1 root root   29K Oct 25  2014 libxext6_2%3a1.3.3-1_amd64.deb
-rw-r--r-- 1 root root  648K Feb 10  2020 libxml2_2.9.4+dfsg1-6.1ubuntu1.3_amd64.deb
-rw-r--r-- 1 root root   19K Dec  6  2016 libxrender1_1%3a0.9.10-1_amd64.deb
-rw-r--r-- 1 root root  6.8K Jul  6 17:58 multiarch-support_2.27-3ubuntu1.2_amd64.deb
-rw-r--r-- 1 root root   50K Feb 28  2018 ucf_3.0038_all.deb
```